stetho/src/main/java/com/facebook/stetho/inspector/elements/android/TextViewDescriptor.java

```diff
@@ -71,5 +95,49 @@ public void afterTextChanged(Editable s) {
         getHost().onAttributeModified(mElement, TEXT_ATTRIBUTE_NAME, s.toString());
       }
     }
+
+    void setAttributeAsText(String text) {
+      // Text will be in the format "attribute1=\"Value 1\" attribute2=\"Value2\""
+      char[] chars = text.toCharArray();
+      String value = "";
+      String key = "";
+      String buffer = "";
+      Map<String, String> keyValuePairs = new HashMap<>();
+      boolean isInsideQuotes = false;
+      for (char c : chars) {
+        if (c == '=') {
+          key = buffer;
+          buffer = "";
+        } else if (c == '\"') {
+          if (isInsideQuotes) {
+            value = buffer;
+            buffer = "";
+          }
+          isInsideQuotes = !isInsideQuotes;
+        } else if (c == ' ' && !isInsideQuotes) {
+          keyValuePairs.put(key, value);
+        } else {
+          buffer += c;
+        }
+      }
+      if (!key.isEmpty() && !value.isEmpty()) {
+        keyValuePairs.put(key, value);
+      }
+
+      for (Map.Entry<String, String> entry : keyValuePairs.entrySet()) {
```

Comment:
regarding my earlier comment about MethodInvoker, I think it's better for now if we just recognize that name == TEXT_ATTRIBUTE_NAME, and call setText() directly